internal/hashing/xxh3_memo_table.go

```diff
@@ -97,14 +106,14 @@ const (
 	loadFactor int64  = 2
 )
 
-func max(a, b uint64) uint64 {
-	if a > b {
-		return a
-	}
-	return b
-}
+// func max(a, b uint64) uint64 {
+// 	if a > b {
+// 		return a
+// 	}
+// 	return b
+// }
 
-var isNan32Cmp = func(v float32) bool { return math.IsNaN(float64(v)) }
+// var isNan32Cmp = func(v float32) bool { return math.IsNaN(float64(v)) }
```

Comment:
Can we remove them instead of commenting out them?

Reply:
bah, i knew i missed something. i'll remove those